pyproject.toml

```
@@ -1,20 +1,87 @@
+[project]
+name = "nipy"
+dynamic = ['version']
+license = {file = "LICENSE"}
+description  = 'A python package for analysis of neuroimaging data'
+readme = 'README.rst'
+classifiers = ["Development Status :: 3 - Alpha",
+               "Environment :: Console",
+               "Intended Audience :: Science/Research",
+               "License :: OSI Approved :: BSD License",
+               "Operating System :: OS Independent",
+               "Programming Language :: Python",
+               "Topic :: Scientific/Engineering"]
+dependencies = [
+    'numpy>=1.22',
+    'scipy>=1.8',
+    'nibabel>=3.2',
+    'sympy>=1.9',
+    'transforms3d'
+]
+
+[[project.maintainers]]
+name = 'nipy developers'
+email = 'neuroimaging@python.org'
+
+[project.urls]
+homepage = 'https://nipy.org/nipy'
+documentation = 'http://nipy.org/nipy/documentation.html'
+source = 'https://github.com/nipy/nipy'
+download = 'https://pypi.org/project/nipy/#files'
+tracker = 'https://github.com/nipy/nipy/issues'
```

IIUC, these will be how they're actually displayed on PyPI. Capitals would look nicer, I think. And you can use `"Several Words" = 'https://...'` if you want to include spaces.

```suggestion
Homepage = 'https://nipy.org/nipy'
Documentation = 'http://nipy.org/nipy/documentation.html'
Source = 'https://github.com/nipy/nipy'
Download = 'https://pypi.org/project/nipy/#files'
Tracker = 'https://github.com/nipy/nipy/issues'
```